app/pods/components/carousel-cards/component.js

```diff
@@ -25,21 +24,26 @@ class carouselCard {
   }
 
   showContent () {
-    $(this.element).find('*').removeClass('display-none')
-    $(this.element).find('*').addClass('display-block')
+    this.element.querySelectorAll(this.element).childNodes.classList.remove('display-none')
```

Comment:
`Failed to execute 'querySelectorAll' on 'Element': '[object HTMLElement]' is not a valid selector.`